src/main/java/com/conveyal/datatools/manager/jobs/MergeProjectFeedsJob.java

```diff
@@ -64,7 +64,7 @@ public void jobLogic () {
         } catch (IOException e) {
             LOG.error("Could not create temp file");
             e.printStackTrace();
-            halt(400, SparkUtils.formatJSON("Unknown error while merging feeds.", 400));
+            halt(400, SparkUtils.formatJSON("Unknown error while merging feeds.", 400, e));
```

Comment:
That makes more sense.  These halts within jobs didn't make much sense to me because they won't affect any HTTP request/response since they happen outside of that scope.